src/main/java/seedu/address/storage/JsonAddressBookStorage.java

```diff
@@ -31,6 +32,30 @@ public Path getAddressBookFilePath() {
         return filePath;
     }
 
+    /**
+     * Similar to {@link #readAddressBook()}.
+     *
+     * @param filePath location of the data. Cannot be null.
+     * @throws DataConversionException if the file is not in the correct format.
+     */
+    public Optional<ReadOnlyHotel> readHotel(Path filePath) throws DataConversionException {
+        requireNonNull(filePath);
+
+        Optional<JsonSerializableHotel> jsonHotel = JsonUtil.readJsonFile(
+                filePath, JsonSerializableHotel.class);
+        if (!jsonHotel.isPresent()) {
+            System.out.println("hotel empty");
+            return Optional.empty();
+        }
+
+        try {
+            return Optional.of(jsonHotel.get().toModelType());
+        } catch (IllegalValueException ive) {
+            logger.info("Illegal values found in " + filePath + ": " + ive.getMessage());
```

Comment:
this should be logger.error right?

Reply:
```suggestion
            logger.error("Illegal values found in " + filePath + ": " + ive.getMessage());
```

Reply:
there seems to be no error method in logger